### ISSUE TITLE
Qt5 upgrade fails on macOS Sierra (10.12)

### ISSUE THREAD

**Issue by marmot1123** [Homebrew/homebrew-science#4333]
### Indicate troubleshooting steps taken
- [x] Ran `brew update` and retried your prior step?
- [x] Ran `brew doctor`, fixed as many issues as possible and retried your prior step?
- [x] Checked that a similar issue hasn't already been filed?
- [x] Looked up the software bug tracker if reporting a build error?
- [x] If you're seeing permission errors tried running `sudo chown -R $(whoami) $(brew --prefix)`?
#### Brief summary of issue:

I updated my Mac's OS from 10.11 to 10.12.
Then I updated and upgraded Homebrew.
But several formulae failed upgrading.

```
$ brew outdated
gcc (5.3.0, 6.1.0) < 6.2.0
haskell-stack (1.0.4, 1.0.4.3, 1.1.2_3) < 1.2.0_1
homebrew/php/php70 (7.0.4, 7.0.6, 7.0.8, 7.0.9) < 7.0.11_2
homebrew/php/php70-mcrypt (7.0.4, 7.0.6, 7.0.8, 7.0.9) < 7.0.11_1
homebrew/science/root (5.34.34_1, 5.34.36) < 5.34.36_1
vim (7.4.1655, 7.4.1795, 7.4.1952, 7.4.1993, 7.4.2033, 7.4.2109) < 8.0.0005
```

Even if I upgraded each formula (for example, `$ brew upgrade gcc`), I failed.
So I `brew doctor`ed and resolved, but I cannot resolve following warning.

```
$ brew doctor
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry and just ignore them. Thanks!

Warning: Some installed formula are missing dependencies.
You should `brew install` the missing dependencies:
  brew install qt5

Run `brew missing` for more details.

$ brew missing
gnuplot: qt5
```

I attempt to install qt5, but I failed.

```
$ brew install qt5
... (log) ...
These open issues may also help:
Update qt5 urls https://github.com/Homebrew/homebrew-core/pull/4905
```

I read the link above but I have no idea what I should do.
#### Link output by `brew gist-logs <formula>`:

```
$ brew gist-logs qt5
```

Result: https://gist.github.com/marmot1123/d05c395618eb8a595e29159278daf75a
#### Output of `brew config`:

```
$ brew config
HOMEBREW_VERSION: 1.0.0
ORIGIN: https://github.com/Homebrew/brew.git
HEAD: a381b597d3ebf40887b50657df269d867a209eb7
Last commit: 18 hours ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 227ea74bb44ec704947828ef1887256fe852fd09
Core tap last commit: 2 hours ago
HOMEBREW_PREFIX: /usr/local
HOMEBREW_REPOSITORY: /usr/local/Homebrew
HOMEBREW_CELLAR: /usr/local/Cellar
HOMEBREW_BOTTLE_DOMAIN: https://homebrew.bintray.com
CPU: octa-core 64-bit haswell
Homebrew Ruby: 2.0.0-p648
Clang: 8.0 build 800
Git: 2.10.0 => /usr/local/bin/git
Perl: /usr/bin/perl
Python: /usr/local/bin/python => /usr/local/Cellar/python/2.7.12/Frameworks/Python.framework/Versions/2.7/bin/python2.7
Ruby: /usr/local/bin/ruby => /usr/local/Cellar/ruby/2.3.1/bin/ruby
Java: 1.8.0_51
macOS: 10.12-x86_64
Xcode: 8.0
CLT: N/A
X11: 2.7.9 => /opt/X11
```

Any advice is appreciated.
Thanks.

**Comment by walsvid** [comment]
Hi, in your error log I find that `gstreamer` maybe missing. How about try `brew info gstreamer` and find out whether you have installed `gstreamer` or not. If you did not install `gstreamer`, I think you can try `brew install gstreamer` first. 


But I find that in the log it said:
```
Support enabled for:
GStreamer .............. no
```
and I do not know why it got error in `Project ERROR: gstreamer-1.0 development package not found`.

**Comment by marmot1123** [comment]
Thanks! Certainly, `gstreamer` was not installed.
Then I tried `brew install gstreamer`.
And then I retried `brew install qt5`, but it happened same thing.
Still I cannot understand why I cannot install qt5.
Anyway, thanks for your comment.

**Comment by ticktricktrack** [comment]
Same problem for me

**Comment by chevinbrown** [comment]
http://stackoverflow.com/questions/39690404/brew-install-qt-does-not-work-on-macos-sierra

**Comment by ilovezfs** [comment]
@chevinbrown that refers to qt not qt5.

**Comment by ilovezfs** [comment]
qt5 should be pouring the bottle.

**Comment by ilovezfs** [comment]
I'm going to assume this is fixed now that qt 5.7.0 has shipped and there is a Sierra bottle. Please let us know if you're still having trouble installing qt5.
